addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -193,3 +194,36 @@ def test_unread_channel_not_unpined_for_operator_after_autovacuum(self):
         with freeze_time(fields.Datetime.to_string(fields.datetime.now() + timedelta(days=1))):
             member_of_operator._gc_unpin_livechat_sessions()
         self.assertTrue(member_of_operator.is_pinned, "unread channel should not be unpinned after autovacuum")
+
+    def test_channel_command_help_in_livechat(self):
+        """Ensures the command '/help' works in a livechat"""
+        channel_info = self.make_jsonrpc_request(
+            "/im_livechat/get_session",
+            {
+                "anonymous_name": "<strong>visitor</strong>",
```

Comment:
why `<strong>` ? remove it?

Reply:
Actually ensure escape/markup is working as intended.